outputter/database.go

```diff
@@ -0,0 +1,91 @@
+package outputter
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aquasecurity/bench-common/check"
+	"github.com/jinzhu/gorm"
+)
+
+// PSGSQL contains the database connection information
+type PSGSQL struct {
+	Host     string
+	User     string
+	Password string
+	SSLMode  string
+	DBName   string
+}
+
+const (
+	// HOST host key
+	HOST = "HOST"
+	// USER key
+	USER = "USER"
+	// PASSWORD key
+	PASSWORD = "PASSWORD"
+	// SSLMODE key
+	SSLMODE = "SSLMODE"
+	// DBNAME key
+	DBNAME = "DBNAME"
+)
+
+// NewPSGSQL constructs a new PSGSQL
+func NewPSGSQL(configValues map[string]string) *PSGSQL {
+	return &PSGSQL{
+		Host:     configValues[HOST],
```

Comment:
do these fields need to be exported? I'd guess we are only using them here in this package